app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -7,63 +7,38 @@
 
     <fragment
         android:id="@+id/nav_home"
-        android:name="com.example.lawuna.ui.home.HomeFragment"
-        android:label="@string/menu_home"
+        android:name="com.example.lawuna.ui.od_camera.CameraActivity"
+        android:label="@string/home_title"
         tools:layout="@layout/fragment_home">
 
         <action
             android:id="@+id/action_HomeFragment_to_HomeSecondFragment"
             app:destination="@id/nav_content" />
     </fragment>
-<!--    <fragment-->
-<!--        android:id="@+id/nav_home_second"-->
-<!--        android:name="com.example.lawuna.ui.home.HomeSecondFragment"-->
-<!--        android:label="@string/home_second"-->
-<!--        tools:layout="@layout/fragment_home_second">-->
-<!--        <action-->
-<!--            android:id="@+id/action_HomeSecondFragment_to_HomeFragment"-->
-<!--            app:destination="@id/nav_home" />-->
-
-<!--        <argument-->
-<!--            android:name="myArg"-->
-<!--            app:argType="string" />-->
-<!--    </fragment>-->
+    <fragment
+        android:id="@+id/nav_home_second"
+        android:name="com.example.lawuna.ui.home.HomeSecondFragment"
+        android:label="@string/home_second"
+        tools:layout="@layout/fragment_home_second">
+        <action
+            android:id="@+id/action_HomeSecondFragment_to_HomeFragment"
+            app:destination="@id/nav_home" />
 
-<!--    <fragment-->
-<!--        android:id="@+id/nav_od_camera"-->
-<!--        android:name="com.example.lawuna.ui.od_camera.CameraActivity"-->
-<!--        tools:layout="@layout/od_activity_camera" />-->
```

Comment:
Check the syntax here

Reply:
I have actually removed all that code since its not being used.